### DISCUSSION TITLE
ModuleNotFoundError: No module named 'numpy'?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#801] [Q&A] [answered]
Before I compile SeisSol with no problem, but I recompile SeisSol I encouter a problem, I `pip3 install numpy`, but there is still a problem with `ModuleNotFoundError: No module named 'numpy'`,Do you kown why this happend?
![image](https://user-images.githubusercontent.com/114474894/220608430-c4e18ca0-cf43-4397-af10-fa5169cfa4bb.png)

There is the detail about my cmake :
https://github.com/buzhiqiuyuebai/errorlog/blob/main/compilererror#:~:text=ModuleNotFoundError%3A%20No%20module%20named%20%27numpy%27

**Comment by krenzland** [comment, marked as answer]
You probably have multiple versions of Python installed and SeisSol's cmake is grabbing a different one. This is also clearly stated in your attached logfile:
`-- Found Python3: /home/ccxmpich/bin/python3.9 (found version "3.9.0") found components: Interpreter `

**Reply by buzhiqiuyuebai** [reply]
> You probably have multiple versions of Python installed and SeisSol's cmake is grabbing a different one. This is also clearly stated in your attached logfile: `-- Found Python3: /home/ccxmpich/bin/python3.9 (found version "3.9.0") found components: Interpreter `

Yes,you are right, I installed python3.9 in `~/bin` before, but when I use `python3 --version`, it shows python3.6.8 in /usr/local/bin, so I installed it in another catalog, but not uninstall python3.9 in `~/bin` , so it caused by this reason. Many thanks to you!
